sdk/cognitiveservices/Vision.Face/src/Generated/PersonDirectoryExtensions.cs

```diff
@@ -39,14 +39,37 @@ public static partial class PersonDirectoryExtensions
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<IList<EnrollmentResponse>> GetPersonsAsync(this IPersonDirectory operations, string start = default(string), string top = default(string), CancellationToken cancellationToken = default(CancellationToken))
+            public static async Task<IList<EnrollmentResponse>> GetPersonsAsync(this IPersonDirectory operations, System.Guid? start = null, string top = default(string), CancellationToken cancellationToken = default(CancellationToken))
             {
                 using (var _result = await operations.GetPersonsWithHttpMessagesAsync(start, top, null, cancellationToken).ConfigureAwait(false))
                 {
                     return _result.Body;
                 }
             }
 
+            /// <summary>
+            /// Retrieve list of person information in person directory.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='start'>
+            /// </param>
+            /// <param name='top'>
+            /// </param>
+            /// <param name='cancellationToken'>
+            /// The cancellation token.
+            /// </param>
+            public static async Task<IList<EnrollmentResponse>> GetPersonsAsync(this IPersonDirectory operations, string start = null, string top = default(string), CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
shouldn't `start` be Guid like the others?

Reply:
just noticed that this is an overload. but it is not clear to me why there are no overloads for other cases of the breaking change from string to Guid